include/ruby/intern.h

```diff
@@ -952,6 +952,7 @@ VALUE rb_path2class(const char*);
 VALUE rb_class_name(VALUE);
 VALUE rb_autoload_load(VALUE, ID);
 VALUE rb_autoload_p(VALUE, ID);
+VALUE rb_autoload_at_p(VALUE, ID, VALUE);
```

Comment:
Could you move this to `internal.h`?
`include/ruby/intern.h` is basically part of the public C-API (yes, it's confusing).

Reply:
Thanks, that was actually an outstanding question in the ruby-lang thread.

Reply:
Fixed.